.gitignore

```diff
@@ -3,3 +3,4 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
+telegraf.conf
```

Comment:
Can you revert this file, I like to keep this trimmed down to only the files the build system creates.

Reply:
Done